src/containers/AlbumsContainer.jsx

```diff
@@ -1,3 +1,33 @@
 import React, { useState, useEffect } from 'rect';
 import Albums from '../components/Albums/Albums';
-import { fetchArtist }
\ No newline at end of file
+import { fetchArtist fetchAlbum } from '../services/ApiFetch';
+import PropTypes from 'prop-types';
+
+const AlbumsContainer = ({ match }) => {
+    const [ artist, setArtist ] = useState({});
+    const [albums, setAlbums ] = useState([]);
+
+    useEffect(() => {
+        fetchArtist(match.params.id)
+        .then(artist => setArtist(artist));
+
+        fetchAlbum(match,params.id)
```

Comment:
not sure if you fixed this typo in a later commit, but you've got a comma instead of a period in match.params.id